docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -261,7 +261,7 @@
             },
             annotations: {
               summary: 'RAID Array is degraded',
-              description: "RAID array '{{ $labels.device }}' on {{ $labels.instance }} is in degraded state due to one or more disks failures. Number of spare drives is insufficient to fix issue automatically.",
+              description: "RAID array '{{ $labels.device }}' at {{ $labels.instance }} is in degraded state due to one or more disks failures. Number of spare drives is insufficient to fix issue automatically.",
```

Comment:
on/at/in? Dunno, but I don't see reason to change on -> at?

Reply:
Other alerts already use at with instance.

Reply:
Yes, consistency is good.